ui/src/addrgbpanel.cpp

```diff
@@ -53,7 +54,9 @@ AddRGBPanel::AddRGBPanel(QWidget *parent, const Doc *doc)
     connect(m_uniCombo, SIGNAL(currentIndexChanged(int)),
             this, SLOT(slotUniverseChanged()));
     connect(m_compCombo, SIGNAL(currentIndexChanged(int)),
-            this, SLOT(slotComponentsChanged(int)));
+            this, SLOT(slotComponentsChanged()));
+    connect(m_16bitCheck, SIGNAL(stateChanged(int)),
```

Comment:
Which Qt version have you used to test this signal? Qt 5.x?
According to documentation, Qt 6.7+ emits [checkStateChanged](https://doc.qt.io/qt-6/qcheckbox.html#checkStateChanged)

Reply:
I just used the defaults, so I think it's Qt 5 (since the GitHub actions seem happy, and their name suggests they use Qt 5). I'm happy to test with Qt 6, but how to build? I checked [the wiki](https://github.com/mcallegari/qlcplus/wiki) but didn't find a page that explains it. (In fact, [this page](https://github.com/mcallegari/qlcplus/wiki/QLC-Plus-5-build-HOWTO) explicitly states that Qt 6 is not supported yet.)

Reply:
In Qt 6.9.0 `stateChanged(int)` is indeed deprecated in favor of `checkStateChanged(Qt::CheckState)`.

Reply:
This should not be 6.9.0, but also not 6.0.0. It should be 6.7.0 as the documentation states it is introduced in that version.